### ISSUE TITLE
Book entries doesn't load

### ISSUE THREAD

**Issue by Alexis-Silvery** [VazkiiMods/Patchouli#642]
### Mod loader

Forge

### Minecraft version

1.18.2

### Patchouli version

1.18.2-71.1

### Modloader version

Forge - 40.1.93

### Modpack info

https://beta.curseforge.com/minecraft/modpacks/vault-hunters-1-18-2

### The latest.log file

https://mega.nz/file/ZgJghYaB#lOSFG-a-lgneE0Goi1xkNZk74AnschlmspnJ7BfugJs

### Issue description

When I enter in the server my book apears the message  "loading error" and give me no entries.
![1](https://user-images.githubusercontent.com/119908477/218054663-5c4228e7-7563-4cf1-9fd5-8f8a905587ea.PNG)
![2](https://user-images.githubusercontent.com/119908477/218054670-7bc1772c-d59b-475b-934b-eebd15f2fa29.PNG)
![3](https://user-images.githubusercontent.com/119908477/218054672-442e8fc0-80c6-4559-ac63-61734ef94031.PNG)

**Comment by williewillus** [comment]
Have you read the log errors/the log message shown in your screenshots?
One of the pages has an invalid item declared.

**Comment by Alexis-Silvery** [comment]
> 

Yes, I have read the log, in the log it says the book recipe is invalid.
But in my friends game, it doesn't show any error.

**Comment by williewillus** [comment]
And the modpack is the exact same between you two with no edits?

**Comment by Alexis-Silvery** [comment]
> And the modpack is the exact same between you two with no edits?

yes

**Comment by Multi-Coder-24** [comment]
Have you checked the json for the entry it mentions which is vault_gear/tool_station page 2. Is the json valid and does the item exist as it is showing that there is an unknown item with the ID of the_vault:jewel

**Comment by Multi-Coder-24** [comment]
I have taken a look through the logs and there are a few errors of interest.
The first one is "Invalid icon item stack: Unknown item ID: the_vault:tool_station"
The second is "Recipe the_vault:tool_station (of type minecraft:crafting) not found"
And the last being "Unknown item ID: the_vault:jewel"
Can you confirm that not only the items exist but the recipe also exists
